LR_Tourist/Tourist/TouristConsole.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The project should have the name `TouristConsoleApp`.
Also, it should be allocated to the `TouristConsoleApp` project.
**Also, all files in this project should have correct namespace (should be fixed in all projects).**

Reply:
Still not resolved.

Reply:
fixed